src/runtime/composables/cwa-resource.ts

```diff
@@ -6,15 +6,15 @@ export type IriProp = {
  iri: string
 }
 
-interface cwaResourceUtilsOps {
+interface CwaResourceUtilsOps {
```

Comment:
This is can be as iri is first parameter passed.
```
CwaResourceUtilsOps {
  manager?: {
    disabled?: boolean
    options?: ManageableComponentOptions
  }
}
```

Reply:
fixed (I guess)

Reply:
Interface is fine, please update all usage in useCwaResource e.g. 2nd argument of useCwaResourceManageable in useCwaResource should be ops?.manager?.options

Reply:
And any code which is calling `useCwaResource` should use this type/object structure now

Reply:
fixed

Reply:
fixed